File: 800/Java/P8.java
import java.util.*;

public class P8 {
    static void solve(Scanner scanner) {
        int n = scanner.nextInt();
        int k = scanner.nextInt();

        HashMap<Integer, Integer>mpp = new HashMap<>();

        for(int i = 0; i < n; i++) {
            int num = scanner.nextInt();
            mpp.put(num, mpp.getOrDefault(num, 0) + 1);
        }

        if(mpp.containsKey(k))  System.out.println("YES");
        else    System.out.println("NO");
    }   
    
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int t = scanner.nextInt();
        while(t-- > 0)  solve(scanner);
        scanner.close();
    }
}

File: 800/C++/p8.cpp
#include<bits/stdc++.h>
using namespace std;

void solve() {
    int n, k;
    cin>>n>>k;

    unordered_map<int, int>mpp;
    for(int i = 0; i < n; i++) {
        int num;
        cin>>num;
        mpp[num]++;
    }

    if(mpp[k])  cout<<"YES"<<endl;
    else    cout<<"NO"<<endl;
}

int main() {
    int t;
    cin>>t;
    while(t--)  solve();
    return 0;
}